CriteoPublisherSdk/Sources/Criteo.m

```diff
@@ -135,6 +137,14 @@ - (void)setUserData:(CRUserData *)userData {
 
 #pragma mark - Bidding
 
+- (NSNumber *)childDirectedTreatment {
+  return self.bidManager.childDirectedTreatment;
+}
+
+- (void)setChildDirectedTreatment:(NSNumber *)childDirectedTreatment {
```

Comment:
nullable

Reply:
I have added nullability specifier to the `getter` and `setter` of `childDirectedTreatment` inside the `Criteo` class.
Resolving this conversation.